frontend/.env.template

```diff
@@ -1 +1,2 @@
-VUE_APP_URL=%%SQUARE_URL%%
+VUE_APP_SQUARE_URL=%%SQUARE_URL%%
+VUE_APP_COOKIE_BOT_ID=36777cf3-0dbd-43eb-b0c5-3b9797f9427e
```

Comment:
Should this be hard coded and checked in? AFAIK, this ID is something specific to our cookie bot account. So maybe it's better to put a placeholder there?

Reply:
Oh sorry, my comment was on the wrong line. I mean the `VUE_APP_COOKIE_BOT_ID`